src/test/resources/propertiesFile/config.Properties

```diff
@@ -4,4 +4,6 @@ url=http://demo.automationtesting.in/Register.html
 
 Please_Enter_Your_Input_down_Here
 
+gender=//*[@id=\"basicBootstrapForm\"]/div[5]/div/label[1]/input 
```

Comment:
Your Xpath is valid, but try to create custom Xpath if possible. The instruction is store input here on properties file.